kitsune/questions/events.py

```diff
@@ -86,8 +84,7 @@ def _make_mail(locale, user, context):
             is_asker = asker_id == user.id
             extra_params = {}
             if is_asker:
-                auth_str = get_auth_str(user)
-                extra_params['auth'] = auth_str
+                extra_params['auth'] = reverse('users.fxa_authentication_init')
```

Comment:
Is this some django magic which redirects users to the url in the `?auth=` parameter if they're not logged in, because I can't see where we're doing this ourselves?